src/parallaximage/ParallaxImage.tsx

```diff
@@ -0,0 +1,260 @@
+// Parallax effect inspired by https://github.com/oblador/react-native-parallax/
+
+import React, { Component } from 'react';
+import {
+    View,
+    Animated,
+    Easing,
+    ActivityIndicator,
+    findNodeHandle,
+    ImageProps,
+    StyleProp,
+    ViewStyle,
+    NativeSyntheticEvent,
+    ImageLoadEventData,
+    ImageErrorEventData
+} from 'react-native';
+import styles from './ParallaxImage.style';
+import type Carousel from 'src/carousel/Carousel';
+
+type ParallaxImageProps = {
+  carouselRef: Carousel<unknown> | null; // passed from <Carousel />
```

Comment:
As I mention above, If we can replace `Unknown` with `any`. Then please change here also.

Reply:
Same answer as Pagination

Reply:
Again, let's use `TData` here as well.